osf_tests/test_analytics.py

```diff
@@ -42,3 +44,125 @@ def test_increment_user_activity_counters(self):
         assert_equal(user.get_activity_points(), 0)
         analytics.increment_user_activity_counters(user._id, 'project_created', date.isoformat())
         assert_equal(user.get_activity_points(), 1)
+
+
+@pytest.fixture()
+def user():
+    return UserFactory()
+
+
+@pytest.fixture()
+def project():
+    return ProjectFactory()
+
+@pytest.fixture()
+def project_with_contrib(user):
```

Comment:
This should be unnecessary. Every project has a creator, so you can just use the `project` fixture and access `project.creator` on it.